internal/wasm/module_test.go

```diff
@@ -293,7 +293,7 @@ func TestModule_Validate_Errors(t *testing.T) {
 		expectedErr string
 	}{
 		{
-			name: "StartSection points to an invalid func",
+			name: "StartSection points to an infunc",
```

Comment:
possibly due to some mechanical replacement
```suggestion
			name: "StartSection points to an invalid func",
```